framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1013,14 +1007,12 @@ private void annotate(
                     stubWarn(
                             "Wildcard type <"
                                     + atype
-                                    + "> doesn't match type in stubs file: <"
+                                    + "> doesn't match type in stubs file"
```

Comment:
Should we in general avoid `doesn't` and other contractions? Not really relevant for this PR, but notice that in a message above it's "does not".